package/kedro_viz/constants.py

```diff
@@ -9,37 +9,6 @@
 DEFAULT_HOST = "127.0.0.1"
 DEFAULT_PORT = 4141
 
-VIZ_DEPLOY_TIME_LIMIT = 60
+VIZ_DEPLOY_TIME_LIMIT = 300
 
-AWS_REGIONS = [
```

Comment:
Yes, we will support AWS deployment. Since S3 is a global resource, we do not need region to push contents to S3. Earlier we used region to generate the hosted link (http://{bucket-name}.s3-website.{region}.amazonaws.com/). Since we are asking for hosted endpoint from the users (to be consistent across cloud providers), region is no longer needed. Thank you